Makefile

```diff
@@ -122,7 +122,7 @@ check: unit
 
 unit:
 	@$(call print, "Running unit tests.")
-	$(GOLIST) | $(XARGS) env $(GOTEST) -test.timeout=20m
+	$(UNIT)
```

Comment:
We need to change the `tarodb` path for migration in the `migrate-xxx` targets (10 lines up or so)

Reply:
Good catch, fixed.